rom/dev/README.md

```diff
@@ -66,26 +66,24 @@ following topics:
 ### 5.1 Initial Device ID (IDEVID) DICE Layer
 
 This layer represents the manufacturer/silicon vendor device identity. This layer's CDI is constructed
-using a deobfuscated UDS. A self signed CSR is generated during (or when requested) cold reset.
-CSR is provided to the manufacturer/silicon vendor CA for issuing a Manufacturer Device Certificate.
+using a deobfuscated UDS. A self signed CSR is generated (if requested) during  cold reset.
+CSR is provided to the manufacturer/silicon vendor CA for issuing a Manufacturer Device Certificate. Please refer to https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md#provisioning-idevid-during-manufacturing for additional details on CSR generation request.
 
 ### 5.2 Local Device ID DICE (LDEVID) Layer
 
 This layer represents the owner identity. This layer's CDI is constructed by mixing some entropy
 once the owner acquires the devices. The primary purpose of this layer is to mitigate supply
 chain attacks via owner providing some entropy via fuses which is further randomized via the
-deobfuscation engine. ROM generates a certificate for this layer. The certificate is signed by
-the IDEVID private key.
+deobfuscation engine. ROM generates a certificate signature for this layer using the IDEVID private key.
 
 ### 5.2 Alias FMC DICE Layer
 
 In traditional DICE architectures, the Alias layer is controlled by the FMC. However, there are no standard
-mechanisms available to tie the Alias Certificate back to the manufacturer. Historically,
-non-standard X509 certificate extensions [CHECK]. The only standard way currently available is for LDEVID
-private key to sign the Alias FMC Certificate. However, LDEVID private key is a critical key
+mechanisms available to tie the Alias Certificate back to the manufacturer. The only standard way currently 
+available is for LDEVID private key to sign the Alias FMC Certificate. However, LDEVID private key is a critical key
 with very limited in-field renewability. Hence LDEVID private key must never leave ROM and must
 be cleared prior to ROM exit. This implies we need to do part of the Alias FMC derivations in ROM,
```

Comment:
I don't think we're doing anything non-standard here. It is normal do generate an alias key and then make it usable by the next layer. LDevID is a device specific key w/o any firmware measurements, so it is expected that it is only wielded by ROM.